webpack.config.js

```diff
@@ -18,6 +18,9 @@ const config = {
     libraryTarget: 'commonjs2',
     devtoolModuleFilenameTemplate: '../[resource-path]',
   },
+
+  node: false, // prevent to include webpack's node polyfill, 📖 -> https://webpack.js.org/configuration/node/
```

Comment:
This is saying we don't need node polyfills because we're running in node?

Reply:
Yes.